### DISCUSSION TITLE
Windows Defended identifies PE as HackTool:Win64/ExplorerPatcher!MTB and removes it

### DISCUSSION THREAD

**Discussion by Volod-D** [valinet/ExplorerPatcher#3256] [Q&A]
Just recently my Defender has tried to remove the PE from my PC. I hope that there is nothing really wrong with this indeed very useful extension, but it would be good to get some evidence. Besides, if the program is safe indeed, is this a way to prevent the Defender from doing this in the future, and to the other users too? 

I would argue that the Patcher's options should be integrated into Windows, otherwise it's design is often stupid and inconvenient for "geek" users like me

**Comment by kenvix** [comment]
The code indicates that they did so intentionally. Fuck Microsoft

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/3228

Please close this.
